packages/sveltekit/test/common/utils.test.ts

```diff
@@ -34,7 +34,7 @@ describe('isHttpError', () => {
     expect(isHttpError(httpErrorObject)).toBe(true);
   });
 
-  it.each([new Error(), redirect(301, '/users/id'), 'string error', { status: 404 }, { body: 'Not found' }])(
+  it.each([new Error(), { status: 301, message: '/users/id' }, 'string error', { status: 404 }, { body: 'Not found' }])(
```

Comment:
calling `redirect` throws since Sveltekit 2.0, so I rewrote this test to what `redirect` used to return in kit 1.x (this is not breaking or relevant for kit 2.0)